src/metadata.js

```diff
@@ -40,6 +40,7 @@ function formatImageMetadata(metadata) {
   }
 
   const imageOrientationSlide = metadata['00480102']['Value'];
+  const frameOfReferenceUID = metadata['00200052']['Value'][0];
```

Comment:
This code is consistent with the rest of the package, but it made me with we didn't have hex everywhere.  I added an issue for this.

https://github.com/dcmjs-org/dicom-microscopy-viewer/issues/27

Reply:
I generally agree with you on the use of keywords over tags. Let's discuss in #27 and address this with another PR, ok?